### ISSUE TITLE
WISH: col/rowCounts but for inequalities

### ISSUE THREAD

**Issue by PeteHaitch** [HenrikBengtsson/matrixStats#134]
We have `rowCounts(x, value = 10)` as an efficient version of `rowSums(x == 10)`. It'd be neat to have similarly efficient versions of `rowSums(x > 10)`, `rowSums(x >= 10)`, `rowSums(x <= 10)`, `rowSums(x < 10)`, and `rowSums(x != 10)`.

Conceptually, it could be it's own function or it might add an argument to `col/rowCounts()`, something like:

```R
rowCounts( x, rows = NULL, cols = NULL, value = TRUE, na.rm = FALSE, 
           dim. = dim(x), comparison = c("==", "<", "<=", ">", ">=", "!="), ...)
```

**Comment by HenrikBengtsson** [comment]
Yes, this would be neat.  It's related https://github.com/HenrikBengtsson/matrixStats/issues/51

**Comment by MLopez-Ibanez** [comment]
This will also provide a way to count non-NA values.
